sdk/go/common/workspace/loaders.go

```diff
@@ -47,6 +46,27 @@ var policyPackProjectSingleton *policyPackProjectLoader = &policyPackProjectLoad
 	internal: map[string]*PolicyPackProject{},
 }
 
+// readFile wraps os.ReadFile and also strips the Byte-order Mark (BOM) if present.
```

Comment:
Nit:

```suggestion
// readFile wraps os.ReadFile and also strips the UTF-8 Byte-order Mark (BOM) if present.
```

It's more ugly, but would it be worth naming the function something like `readFileStripUTF8BOM`?